ethpm_cli/parser.py

```diff
@@ -108,6 +110,34 @@ def add_alias_arg_to_parser(parser: argparse.ArgumentParser) -> None:
     )
 
 
+def add_manifest_path_to_parser(
+    parser: Union[argparse.ArgumentParser, argparse._MutuallyExclusiveGroup],
+    help_msg: str,
+) -> None:
+    parser.add_argument(
+        "--manifest-path",
+        dest="manifest_path",
+        action="store",
+        type=Path,
+        help=help_msg,
+    )
+
+
+def add_package_name_and_package_version_to_parser(
```

Comment:
Would you mind splitting this into two different util functions `add_package_name_to_parser()` and `add_package_version_to_parser()`? I can't think off the top of my head a use case where we'd need one and not the other, but it seems cleaner to separate them.